draft-ietf-core-dns-over-coap.md

```diff
@@ -211,6 +211,10 @@ DNS information is called a "DoC client".
 The term "constrained nodes" is used as defined in {{-constr-nodes}}.
 {{-core-link-format}} describes that "Constrained RESTful Environments (CoRE)" realize the Representational State Transfer (REST) architecture {{REST}} in a suitable form for such constrained nodes.
 
+If the DoC server provides Observe capabilities, it is a "subject" as defined in {{-coap-observe, Section 1.2}}.
+This includes administering a "list of observers".
```

Comment:
```suggestion
A DoC server can provide Observe capabilities as defined in {{-coap-observe, Section 1.2}}.
As part of that, it administers a "list of observers".
```

The "subject" term is present in RFC7641, but I have never encountered it in any later literature, implementations, or spoken use.